### PR TITLE
Update Stale bot message

### PR DESCRIPTION
**Pull request by 3d-gussner**

### CODE REVIEW

**Review by leptun — APPROVED** [review]

**Comment by 3d-gussner** [comment]
@leptun Thanks for the approval. Waiting for approval from @Prusa-Support before merging it.

**Review by gudnimg — APPROVED** [review]

**Comment by Prusa-Support** [comment]
Looks good. Thank you.

**Pull request merged**
